java/api/src/main/java/org/ray/api/id/ObjectId.java

```diff
@@ -39,12 +50,66 @@ private static ObjectId genNil() {
   /**
    * Generate an ObjectId with random value.
    */
-  public static ObjectId randomId() {
+  public static ObjectId fromRandom() {
     byte[] b = new byte[LENGTH];
     new Random().nextBytes(b);
     return new ObjectId(b);
   }
 
+  /**
+   * Compute the object ID of an object put by the task.
+   */
+  public static ObjectId forPut(TaskId taskId, int putIndex) {
+    return forPut(taskId, putIndex, TransportType.STANDARD);
+  }
+
+  /**
+   * Compute the object ID of an object put by the task.
+   */
+  public static ObjectId forPut(TaskId taskId, int putIndex, TransportType transportType) {
+    short flags = 0;
+    flags = setIsTaskFlag(flags, true);
```

Comment:
what are the scenarios that `is_task` flag should be set to `false`?  I assume in that case the random `task_id` part should be set to all 1s, but in that case we can simply tell from this, without the need to have a `is_task` flag?

Reply:
I found this is used for `LocalMode`. 

BTW,  @stephanie-wang ,do you need `is_task` flags for something?